### ISSUE TITLE
On the partition of data sets

### ISSUE THREAD

**Issue by xinsheng44** [lvguofeng/GNN_PPI#2]
Hello, I see that your code verification set and test set are divided into the same result, doesn't that mean that the result of the final paper is the best result on the verification set? According to the general data set, it should be training set, verification set, test set, and verification set and test set are not the same. Am I wrong about this or is this the way the job is supposed to be done.

**Comment by lvguofeng** [comment]
Yes, the code we show does not strictly divide into training set, validation set, and test set. Based on our division strategy, we think it is not necessary to divide the test set. Our experimental results show not the optimal results, but the results after training are stable.

It is worth noting that the contribution of our paper is that apart from proposing a new division strategy, it is more important to show the superiority of the graph model in PPI prediction. Compared with the previous methods, it is an absolute lead, even the worst result after the training is stable.

Of course, if you want to study graph model, I think it is necessary to further divide the test set into a validation set and a test set.

**Comment by xinsheng44** [comment]
> Yes, the code we show does not strictly divide into training set, validation set, and test set. Based on our division strategy, we think it is not necessary to divide the test set. Our experimental results show not the optimal results, but the results after training are stable.
> 
> It is worth noting that the contribution of our paper is that apart from proposing a new division strategy, it is more important to show the superiority of the graph model in PPI prediction. Compared with the previous methods, it is an absolute lead, even the worst result after the training is stable.
> 
> Of course, if you want to study graph model, I think it is necessary to further divide the test set into a validation set and a test set.

Thank you very much for your reply
